Initialize board

cbohara opened the pull request:
3rd times a charm? ;)

I removed user_id and added color attributes to pieces

Initializing pieces to start game based on this update in schema

hathcode commented:
Hey Charlie, my merge already removed the user_id field from game table, so no need for you to do that.  You may need to do some conflict resolution or just modify yours and recommit.

cbohara commented:
Hey Kevin I think we should be ok.  I removed the user_id and added the color field in the **pieces** table in this commit.  Does that make sense?  If so, would you be willing to give me a thumbs up?

hathcode commented:
Yeah, I caught that after I looked again. Thought at first you were repeating what I did, sorry.  This does show that you have some conflicts, did you already resolve them?  I have to run here in a few, so I don't know if I'll have the time till later tonight to look over it well enough to give a thumbs up, but if someone else doesn't first, I will soon as I can!
